File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/singlepost/post/ImagesItem.java
package io.github.gusandrianos.foxforreddit.data.models.singlepost.post;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class ImagesItem{

    @SerializedName("resolutions")
    private List<ResolutionsItem> resolutions;

    @SerializedName("source")
    private Source source;

    @SerializedName("variants")
    private Variants variants;

    @SerializedName("id")
    private String id;

    public List<ResolutionsItem> getResolutions(){
        return resolutions;
    }

    public Source getSource(){
        return source;
    }

    public Variants getVariants(){
        return variants;
    }

    public String getId(){
        return id;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/ImagesItem.kt
package io.github.gusandrianos.foxforreddit.data.models

import com.google.gson.annotations.SerializedName

class ImagesItem {
    @SerializedName("resolutions")
    val resolutions: List<ResolutionsItem>? = null

    @SerializedName("source")
    val source: Source? = null

    @SerializedName("id")
    val id: String? = null
}